### PR TITLE
Fix:#2054: Add Changelog Release Notes for 1.14

### LINKED ISSUE #2054: Changelog Release Notes for 1.14

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

Fixes #2054

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`

## Test Plan:

N/A

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: extra line that needs to be removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+## [1.14](https://github.com/brave/brave-ios/releases/tag/v1.14)
+
+ - Added download preview option. ([#1467](https://github.com/brave/brave-ios/issues/1467))
+ - Added user confirmation when external links try to switch apps. ([#551](https://github.com/brave/brave-ios/issues/551))
+ - Added time out details for incorrect pin-code lockout. ([#443](https://github.com/brave/brave-ios/issues/443))
+ - Switching to "Private Browsing Only" closes normal tabs and delete sessions. ([#580](https://github.com/brave/brave-ios/issues/580))
+ - Fixed rewards button from showing up when disabled in settings. ([#1954](https://github.com/brave/brave-ios/issues/1954))
+ - Fixed browser crash when attempting to clear private data on iPad-OS 13.2. ([#1951](https://github.com/brave/brave-ios/issues/1951))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I don't think `iPad-OS` is the correct way if describing iOS. Should use something along the lines of the following:

- Fixed browser crash when clearing private data on iPad running iOS 13.2. ([#1951](https://github.com/brave/brave-ios/issues/1951))

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+ - Fixed AppStore button functionality when AppStore links are opened in the browser. ([#1941](https://github.com/brave/brave-ios/issues/1941))
+ - Fixed Bluetooth keyboard issue with the browser. ([#1146](https://github.com/brave/brave-ios/issues/1146))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Doesn't really explain what the issue was other than mentioning a bluetooth keyboard. Maybe something along the following would explain what was fixed. Doesn't need to be specific but should mention keyboard shortcuts.

 - Fixed several keyboard shortcuts when using a Bluetooth keyboard  . ([#1146](https://github.com/brave/brave-ios/issues/1146))

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+
+## [1.14](https://github.com/brave/brave-ios/releases/tag/v1.14)
+
+ - Added download preview option. ([#1467](https://github.com/brave/brave-ios/issues/1467))
+ - Added user confirmation when external links try to switch apps. ([#551](https://github.com/brave/brave-ios/issues/551))
+ - Added time out details for incorrect pin-code lockout. ([#443](https://github.com/brave/brave-ios/issues/443))
+ - Switching to "Private Browsing Only" closes normal tabs and delete sessions. ([#580](https://github.com/brave/brave-ios/issues/580))
```
Comment (kjozwiak) [inline CHANGELOG.md:8]: nit: maybe adding `deletes current sessions`? Example:

 - Switching to "Private Browsing Only" closes normal tabs and deletes current sessions. ([#580](https://github.com/brave/brave-ios/issues/580))

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
...
+ - Fixed rewards button from showing up when disabled in settings. ([#1954](https://github.com/brave/brave-ios/issues/1954))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Using `being displayed` sounds a bit better than `showing up`:

 - Fixed rewards button being displayed even when disabled in settings. ([#1954](https://github.com/brave/brave-ios/issues/1954))

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,18 @@
...
+ - Fixed browser crash when attempting to clear private data on iPad-OS 13.2. ([#1951](https://github.com/brave/brave-ios/issues/1951))
+ - Fixed AppStore button functionality when AppStore links are opened in the browser. ([#1941](https://github.com/brave/brave-ios/issues/1941))
+ - Fixed Bluetooth keyboard issue with the browser. ([#1146](https://github.com/brave/brave-ios/issues/1146))
+ - Fixed force touch(3D) cursor movement in the URL bar when URL is highlighted. ([#1081](https://github.com/brave/brave-ios/issues/1081))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: I think Apple uses 3D Touch --> https://developer.apple.com/ios/3d-touch/. So maybe something along the lines of the following:

 - Fixed force 3D Touch cursor movement in the URL bar when URL is highlighted. ([#1081](https://github.com/brave/brave-ios/issues/1081))

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (srirambv) [reply CHANGELOG.md]: Apple calls it iPadOS (https://www.apple.com/ca/ipados/). Will remove the extra `-`

File: CHANGELOG.md (hunk 4 above)
Comment (srirambv) [reply CHANGELOG.md:8]: Adding current would make it sound only recent one and not all sessions hence kept it as delete session

**Review by kjozwiak — APPROVED** [review]
Went through changes, LGTM 👍

**Pull request merged**